### ISSUE TITLE
Populate the patient_id/place_id fields within a form group

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#1408]
Patient and placed id fields in a form can be [prepopulated](https://github.com/medic/medic-webapp/blob/dc73dcbfb08edae03434dcaf7b6bd71533a65e66/static/js/controllers/contacts-report.js#L15) (#1266) and hidden (#1189), but the result is a blank page with Prev/Next buttons. To avoid this situation it would be useful to _also_ populate a patient_id/place_id field within a group. We could then show the whole group in a page with the `field-list` appearance; hiding the id field would not affect the form's flow.

Alternatively we can fix the hidden field so it hides the whole page, but from discussion with @garethbowen it seems more advantageous to map data to fields in the form.

**Comment by garethbowen** [comment]
The alternative solution should come cheaply once #1211 is done.

**Comment by abbyad** [comment]
I have a solution that works for now. As suggested by @alxndrsn, we can make use of the "difference between grouping data in the model, and grouping them in the view". I don't think this is possible using XMLform, but the XML can be hacked. 

To avoid doing this manually I am using the following in a post convert script:
```
form='pregnancy'
      
# medic-webapp populates patient_id & place_id fields in model's top level only
# remove `<patient_id/>`
sed -i -r -e 's/<patient_id\/>//g' ${form}.xml

# replace `</pregnancy>` with `<patient_id/></pregnancy>`
sed -i -r -e "s/<\/${form}>/<patient_id\/><\/${form}>/g" ${form}.xml
sed -i -r -e "s/<\/${form}>/<place_id\/><\/${form}>/g" ${form}.xml

# replace `/*/patient_id` with `/patient_id`
sed -i -r -e 's/\/\w+\/patient_id/\/patient_id/g' ${form}.xml
sed -i -r -e 's/\/\w+\/place_id/\/place_id/g' ${form}.xml
```

**Comment by abbyad** [comment]
Closing as this is not needed now. We can open a new issue if this is needed for another reason at a later time.
